crates/diesel_models/src/payment_intent.rs

```diff
@@ -134,8 +134,9 @@ pub enum PaymentIntentUpdate {
         order_details: Option<Vec<pii::SecretSerdeValue>>,
         metadata: Option<pii::SecretSerdeValue>,
     },
-    PaymentAttemptUpdate {
+    PaymentAttemptAndAttemptCountUpdate {
```

Comment:
Should we rename this as `AutoRetryAttemptUpdate` and below variant as `ManualRetryAttemptUpdate` ? I guess there are no other use cases for these structs

Reply:
But these names imply what are we updating, so this would be better even if it's used anywhere else in future right!